<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="arithmetic_group_box">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>261</width>
      <height>801</height>
     </rect>
    </property>
    <property name="title">
     <string>Arithmetic</string>
    </property>
    <widget class="QPushButton" name="average_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Average</string>
     </property>
    </widget>
    <widget class="QPushButton" name="function_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>f(x, y) - f(x - 1, y)</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="basic_arithmetic_grid_layout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="mul_radio_button">
        <property name="text">
         <string>Multiplication</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="add_or_sub_radio_button">
        <property name="text">
         <string>Addition/Subtraction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="value_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="value_slider">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="raw_image_group_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="title">
     <string>Raw Image</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="raw_image">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="raw_histogram_vertical_layout"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="processed_image_group_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="title">
     <string>Processed Image</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="processed_image">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="processed_histogram_vertical_layout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File (&amp;F)</string>
    </property>
    <addaction name="open_file_action"/>
    <addaction name="exit_app_action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="open_file_action">
   <property name="text">
    <string>Open (&amp;O)</string>
   </property>
  </action>
  <action name="exit_app_action">
   <property name="text">
    <string>Exit (&amp;E)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
